### DISCUSSION TITLE
i can't connect to emulator

### DISCUSSION THREAD

**Discussion by zainurihasan** [zepp-health/zeppos-forum#309] [General]
i just set up the dev env... when i run the hello world the app is not running on the emulator, is there any configuration that I am missing?

![Screenshot 2024-02-20 141636](https://github.com/zepp-health/zeppos-forum/assets/15233848/11710933-beec-411b-9ee3-7d3a75675935)

**Comment by MyoungXUE** [comment]
@zainurihasan 
Hello, please provide more information, the app.json information of the hello-world project, and a screenshot of the Simulator window.
